js/localstorage.js

```diff
@@ -0,0 +1,34 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.querySelector('form');
+  const inputs = form.querySelectorAll('input, textarea');
+
+  // Function to save data to local storage
+  function saveData() {
+    const data = {};
+    inputs.forEach((input) => {
+      data[input.name] = input.value;
+    });
+    localStorage.setItem('formData', JSON.stringify(data));
+  }
```

Comment:
- [ ] As you may notice, when saving data in the localStorage and reloading the browser, the email is not saved. and while retrieving the data as you may see, the message is filling the email input. Kindly make sure you fix that.
![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/97100091/0f13887f-1397-4247-abaf-e50f4da9722e)

Reply:
Thank you very much for your review, I have already made all the suggested changes.👍